revolv/static/home.scss

```diff
@@ -99,6 +149,127 @@ body {
     height: 0px !important;
 }
 
+// --- Current Project Styles ------------------------------------------------------------
+
+.current-project {
+    position: relative;
+    width: 75%;
+    height: 450px;
+    padding: 30px;
+    box-sizing: border-box;
+    color: white;
+
+    a {
+        color: white;
+    }
+
+    h1 {
+        color: white;
+    }
+
+    .current-project-inset {
+        width: 50%;
+        padding: 30px;
+        box-sizing: border-box;
+        white-space: pre-wrap;
+    }
+
+    // Progress circle styles
+    .graphics-container {
+        position: absolute;
+        width: 250px;
+        height: 250px;
+        top: calc(50% - 125px);
+        right: 60px;
+
+        .svg-graphics-container {
+            position: absolute;
+            height: 100%;
+            width: 100%;
+            top: 0;
+        }
+
+        .internal-graphics-container {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            transform: translate(-50%, -50%);
+            -moz-transform: translate(-50%, -50%);
+            -webkit-transform: translate(-50%, -50%);
+        }
+
+        .project-badge-line, .project-badge-circle {
+          fill: none;
+        }
+
+        .project-badge-line {
+          stroke: $revolv-blue;
+        }
+
+        .project-badge-circle {
+          stroke: white;
+        }
+
+        .text-container {
+            position: absolute;
+            // margin-left: 25%;
+            left: 50%;
+            top: 50%;
+            transform: translate(-50%, -50%);
+            -moz-transform: translate(-50%, -50%);
+            -webkit-transform: translate(-50%, -50%);
+            height: 220px;
+            width: 220px;
+            color: $revolv-dark-blue;
+
+            .percentage-text {
+                position: absolute;
+                left: 50%;
+                top: 45%;
+                transform: translate(-50%, -50%);
+                -moz-transform: translate(-50%, -50%);
+                -webkit-transform: translate(-50%, -50%);
+            }
+
+            .funded {
+                position: absolute;
+                font-weight: 800;
+                left: 50%;
+                top: 60%;
+                transform: translate(-50%, -50%);
+                -moz-transform: translate(-50%, -50%);
+                -webkit-transform: translate(-50%, -50%);
+            }
+        }
+    }
+
+    .overlay {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        background: rgba(0, 0, 0, .5);
+    }
+}
+
+.text-with-divider {
```

Comment:
This could use a name that represents what it actually is I think, haha. Text with divider is pretty generic